File: MB-Whatsapp/observability/telemetry-policy.ts
export type TelemetryPolicy = {
  maxPayloadKeys: number;
  maxStringLength: number;
  redactPhi: boolean;
  allowExternalExport: boolean;
};

export const DEFAULT_TELEMETRY_POLICY: TelemetryPolicy = {
  maxPayloadKeys: 50,
  maxStringLength: 500,
  redactPhi: true,
  allowExternalExport: false,
};

File: MB-Chat/observability_py/telemetry_policy.py
from dataclasses import dataclass


@dataclass(frozen=True, slots=True)
class TelemetryPolicy:
    max_payload_keys: int = 50
    max_string_length: int = 500
    redact_phi: bool = True
    allow_external_export: bool = False


DEFAULT_TELEMETRY_POLICY = TelemetryPolicy()
